AnnoyancesFilter/Popups/sections/popups_specific.txt

```diff
@@ -6145,6 +6145,9 @@ afyonhaber.com,tamindir.com,bigfishgames.de,adapostasi.com##div[id^="simplemodal
 temaretik.com##div[ng-if="$root.joinGuest"]
 techblogng.com##iframe.HB-Takeover[src="about:blank"]
 com-magazin.de##w-div[role$="dialog"]
+circleofcricket.com#$#body { overflow: visible!important; }
```

Comment:
this is an ad, should be moved in `BaseFilter/sections/general_extensions.txt` CSS section with comment (link to the issue)